website/client/src/components/ui/filterGroup.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div>
+    <h3>
+      {{ title }}
+    </h3>
+    <div class="form-group">
+      <slot></slot>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'filterGroup',
+  props: ['title'],
+};
+</script>
+
+<style lang="scss" scoped>
+  @import '~@/assets/scss/colors.scss';
+
+  h3 {
+    font-family: Roboto, sans-serif;
```

Comment:
font family and letter spacing are not necessary

Reply:
font family is needed, h3 has usually condensed , but the new sidebar h3 needs the normal Roboto :)

Reply:
@Tressley are we completely removing the Roboto condensed?

Reply:
Not completely. It’s still on h1 and h2.